### PR TITLE
Add missing iconv module

### PR DESCRIPTION
**Pull request by prymag**
[BJ-1886] Add missing iconv module

[Jira Ticket](https://bonjoro.atlassian.net/browse/BJ-1886)

#### Features

-   Item 1

#### Testing instructions

-   [ ] Step 1 ...

#### Analytics update required

-   [ ] Yes
-   [ ] No

#### List Updates

-   [ ] Change existing / new tracking etc...

#### Translation updates required?

-   [ ] Yes
-   [ ] No

#### GDPR updates required?

-   [ ] Yes
-   [ ] No

[BJ-1886]: https://bonjoro.atlassian.net/browse/BJ-1886

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,5 +13,8 @@ RUN apk add git $BUILD_TOOLS && \
     docker-php-ext-configure gd --with-freetype --with-jpeg && \
     docker-php-ext-install -j$(nproc) $DOCKER_EXT
 
+RUN apk add gnu-libiconv=1.15-r3 --update-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.13/community/ --allow-untrusted
```
Comment (deevus) [inline Dockerfile:16]: Did you test `gnu-libiconv` from the default repository?

Comment (deevus) [reply Dockerfile:16]: You didn't really explain in your commit why you had to do what you did here. If you come back in a years time you won't remember why you did it. 

Also, you need to syncronise this with the other Dockerfiles in php72 and php73 directories.

**Review by deevus — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (deevus) [reply Dockerfile:16]: I back-tracked our conversation on Slack. I think this is a much better implementation:

https://github.com/craftcms/docker/pull/35

Comment (prymag) [reply Dockerfile]: I updated my  commit. Please see 8d372c72df2ae05d0f11f8158655c5b804f39451.

I was not able to fully test the app on php72 - 73 as there were language features that aren't available on them. Though I tested the `iconv` function and it works.

**Review by NeilDing666 — APPROVED** [review]
LGTM!

**Review by deevus — APPROVED** [review]

**Pull request merged**